.gitignore

```diff
@@ -1,2 +1,5 @@
 /node_modules
-
+test.md
+MIT.md
+package-lock.json
+package.json
```

Comment:
- [ ] Great job adding a `.gitignore` file, and  including node_modules and `test.md` into the file as required. However I would suggest you to remove the rest of the files from your `.gitignore` as these files are required for your project to work properly and if you put them in gitignore they won't be pushed to the repository.  For this project your `.gitignore` should have only two files, for example:

<kbd>
<img width="657" alt="Screenshot 2023-04-26 at 7 33 34 AM" src="https://user-images.githubusercontent.com/44798044/234451878-c7a69b0c-6409-449f-917d-bb4b98d84e4c.png">
</kbd>